cl/assets/react/DisclosureViewer.tsx

```diff
@@ -0,0 +1,463 @@
+                  return (
+                    <tr key={entry.id} className={''}>
+                      <td>
+                        <a title={'Go to PDF'} href={url + '#page=' + entry.page_number}>
+                          <i className="fa fa-file-text-o gray" />
+                        </a>
+                        &nbsp;
+                        {entry.redacted == true ? (
+                          <i title={'Redaction present in row'} className="fa fa-file-excel-o black" />
+                        ) : (
+                          ''
+                        )}
+                      </td>
+                      {is_admin == true ? (
+                        <td>
+                          <a href={`/admin/disclosures/${key.replaceAll('_', '').slice(0, -1)}/${entry.id}/`}>
+                            <i className="fa fa-pencil gray"></i>
+                          </a>
+                        </td>
+                      ) : (
+                        ''
+                      )}
+
+                      {Object.entries(fields).map(([key, value]) => {
+                        return (
+                          <td key={key}>
+                            {convertTD(entry[value], title, value)}
+                            {entry[value] == -1 ? <i className="fa fa-eye-slash black" /> : ''}
+                          </td>
+                        );
+                      })}
+                    </tr>
+                  );
+                })}
+            </tbody>
+          </Table>
+        </div>
+      ) : (
+        ''
+      )}
+    </div>
+  );
+};
+
+const Support = () => {
+  return (
+    <div id={'support'}>
+      <h3>
+        <span>
+          Support FLP <i className="fa fa-heart-o red pull-right" />
+        </span>
+      </h3>
+      <p className="v-offset-above-1">
+        CourtListener is a project of{' '}
+        <a href="https://free.law" target="_blank" rel="noreferrer">
+          Free Law Project
+        </a>
+        , a federally-recognized 501(c)(3) non-profit. We rely on donations for our financial security.
+      </p>
+      <p>Please support our work with a donation.</p>
+      <p>
+        <a href="" className="btn btn-danger btn-lg btn-block">
+          Donate Now
+        </a>
+      </p>
+    </div>
+  );
+};
+
+const Thumb = (data: Data) => {
+  return (
+    <div className="v-offset-below-4">
+      <h3>
+        <span>
+          Download
+          <a href={`/api/rest/v3/financial-disclosures/?id=${data.id}`}>
+            <i className="fa fa-code gray pull-right" />
+          </a>
+        </span>
+      </h3>
+      <hr />
+      <a href={data.filepath}>
+        {data.thumbnail != null ? (
+          <img
+            src={data.thumbnail}
+            alt="Thumbnail of Judge Portrait"
+            height={'150'}
+            className="img-responsive thumbnail shadow img-thumbnail judge-pic"
+          />
+        ) : (
+          <div
+            height={'150'}
```

Comment:
I...don't think you can put a height here, can you?

Reply:
hmm... how about width.

Reply:
I...don't think so?